server/container/guice/jpa-guice/src/test/java/org/apache/james/JPAJamesServerTest.java

```diff
@@ -80,14 +80,18 @@ public void jpaGuiceServerShouldUpdateQuota() throws Exception {
         smtpMessageSender.connect(LOCALHOST, SMTP_PORT)
             .sendMessageWithHeaders(USER, USER, "header: toto\\r\\n\\r\\n" + Strings.repeat("0123456789\n", 1024))
             .awaitSent(AWAIT);
+        AWAIT.until(() -> imapMessageReader.connect(LOCALHOST, IMAP_PORT)
+            .login(USER, PASSWORD)
+            .select(IMAPMessageReader.INBOX)
+            .hasAMessage());
 
         assertThat(
             imapMessageReader.connect(LOCALHOST, IMAP_PORT)
                 .login(USER, PASSWORD)
                 .getQuotaRoot(IMAPMessageReader.INBOX))
             .isEqualTo("* QUOTAROOT \"INBOX\" #private&toto@james.local\r\n" +
                 "* QUOTA #private&toto@james.local (STORAGE 12 50)\r\n" +
-                "AAAB OK GETQUOTAROOT completed.\r\n");
+                "AAAE OK GETQUOTAROOT completed.\r\n");
```

Comment:
how do you know what the tag should be ?

Reply:
prescience. Or maybe that fact that there are 5 imap operations done.
but maybe I can replace isEqualTo be startsWith

Reply:
yes we can